wifiphisher/common/interfaces.py

```diff
@@ -243,6 +243,36 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def _generate_random_address(self):
+        """
+        Make and return the randomized MAC address
+
+        :param self: A NetworkManager object
+        :type self: NetworkManager
+        :return: A MAC address
+        :rtype str
+        """
+
+        mac_addr = "02:00:00:{:02x}:{:02x}:{:02x}".format(random.randint(0, 255),\
```

Comment:
The OUI part can be hard-coded. According to <a href="https://upload.wikimedia.org/wikipedia/commons/9/94/MAC-48_Address.svg">this</a>, the "02:00:00" part means that the first octet "00000010", so the address is locally administered and unicast which is not really important in our case.

Wi-Fi Pineapple uses the hardcoded OUI "00:13:37" even though it's registered by "Orient Power Home Network Ltd". I believe we can use "00:00:00". It should be fine in all cases.

Whatever we use, it is important to put it in `constants` file and call it from there. I've seen many WIDS detecting the use of a Pineapple by looking for the "00:13:37". We want to change this easily if needed.